ansible_builder/steps.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, context_file):
         self.steps.append("ADD {0} /build/".format(context_file))
         container_path = os.path.join('/build/', context_file)
         self.steps.append(
-            "RUN pip3 install --upgrade -r {content}".format(content=container_path)
+            "RUN pip3 install --no-cache-dir --upgrade -r {content}".format(content=container_path)
```

Comment:
So, we have a few options here. I think we could expect users to setup their own pip.conf file, which is fair. Or be more aggressive here, in ansible-builder.

I'm not sure the best option honestly, if we start config file settings here, then we are on the hook for more pip settings in the future.